apps/web/core/components/features/task/task-status.tsx

```
@@ -1023,147 +1031,121 @@
+										item?.className
 									)}
+								/>
+								{isSelected && issueType !== 'issue' && (
+									<button
+										onClick={(e) => {
+											e.stopPropagation();
+											if (onChange && multiple && item_value) {
+												const newValues = values.filter((v) => v !== item_value);
+												onChange(newValues.join(','));
+											}
+											onRemoveSelected && onRemoveSelected();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)